src/js/support/mobile/widget/SearchBar.js

```diff
@@ -662,17 +654,9 @@
 			function inputBlur(self) {
 				var ui = self._ui,
 					inputtedText = ui.input.value,
```

Comment:
'inputtedText' is assigned a value but never used

Reply:
Good catch. Thanks.